src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppAppRunner/AppStack.cs

```diff
@@ -0,0 +1,130 @@
+                        ImageConfiguration = new CfnService.ImageConfigurationProperty
+                        {
+                            Port = settings.Port.ToString(),
+                            StartCommand = !string.IsNullOrWhiteSpace(settings.StartCommand) ? settings.StartCommand : null
+                        }
+                    }
+                }
+            };
+
+            if(!string.IsNullOrEmpty(settings.EncryptionKmsKey))
+            {
+                var encryptionConfig = new CfnService.EncryptionConfigurationProperty();
+                appRunnerServiceProp.EncryptionConfiguration = encryptionConfig;
+
+                encryptionConfig.KmsKey = settings.EncryptionKmsKey;
+            }
+
+            var healthCheckConfig = new CfnService.HealthCheckConfigurationProperty();
+            appRunnerServiceProp.HealthCheckConfiguration = healthCheckConfig;
+
+            healthCheckConfig.HealthyThreshold = settings.HealthCheckHealthyThreshold;
+            healthCheckConfig.Interval = settings.HealthCheckInterval;
+            healthCheckConfig.Protocol = settings.HealthCheckProtocol;
+            healthCheckConfig.Timeout = settings.HealthCheckTimeout;
+            healthCheckConfig.UnhealthyThreshold = settings.HealthCheckUnhealthyThreshold;
+
+            if(string.Equals(healthCheckConfig.Protocol, "HTTP"))
+            {
+                healthCheckConfig.Path = string.IsNullOrEmpty(settings.HealthCheckPath) ? "/" : settings.HealthCheckPath;
+            }
+
+            var instanceConfig = new CfnService.InstanceConfigurationProperty();
+            appRunnerServiceProp.InstanceConfiguration = instanceConfig;
+
+            IRole role;
+            if (settings.ApplicationIAMRole.CreateNew)
+            {
+                role = new Role(this, "TaskRole", new RoleProps
+                {
+                    AssumedBy = new ServicePrincipal("tasks.apprunner.amazonaws.com")
+                });
+            }
+            else
+            {
+                if (string.IsNullOrEmpty(settings.ApplicationIAMRole.RoleArn))
+                    throw new InvalidOrMissingConfigurationException("The provided Application IAM Role ARN is null or empty.");
+
+                role = Role.FromRoleArn(this, "TaskRole", settings.ApplicationIAMRole.RoleArn, new FromRoleArnOptions
+                {
+                    Mutable = false
+                });
+            }
+            instanceConfig.InstanceRoleArn = role.RoleArn;
+
+            instanceConfig.Cpu = settings.Cpu;
+            instanceConfig.Memory = settings.Memory;
+
+            var service = new CfnService(this, "AppRunnerService", appRunnerServiceProp);
+
+            var output = new CfnOutput(this, "EndpointURL", new CfnOutputProps
+            {
+                Value = $"https://{service.AttrServiceUrl}/"
+            });            
```

Comment:
I think it it still has value for users that are looking at the CloudFormation stack in console or toolkit. Currently you only see the displayed resources value after a deployment. Beanstalk I'm okay removing because we could only get it to show the IP address from CF outputs..